client/src/components/ProfileEdit/data.js

```diff
@@ -0,0 +1,6 @@
+export const genders = [
+	'Male',
+	'Female',
+	'Non-Binary',
```

Comment:
Keep in mind when integrating that in the backend these are lower-case and there's also the option "others".